src/main/webapp/index.xhtml

```diff
@@ -36,25 +36,32 @@
                 <h:message id="trade-err" class="error" for="trade-btn"/>
                 <h:message id="errmsg" class="error" for="bcode"/>
                 <h:outputText id ="totalPrice" value = "#{localTradeList.totalValue}"/>
-            </h:form>
             <!-- dataTable instead of ui:repeat for clear and easy table -->
-            <h:dataTable id="trade-list" 
-                         value="#{localTradeList.items}" 
-                         var="item"
-                         styleClass="trade-table hidden"
-                         style="display:#{localTradeList.items.isEmpty() ? 'none;' : 'block'}">
-                <h:column>
-                    <f:facet name="header">Barcode</f:facet>
-                    #{item.barcode}
-                </h:column>
-                <h:column>
-                    <f:facet name="header">Item</f:facet>
-                    #{item.name}
-                </h:column>
-                <h:column>
-                    <i class="far fa-trash-alt"></i>
-                </h:column>
-            </h:dataTable>
+            	<h:dataTable id="trade-list" 
+                         	value="#{localTradeList.items}" 
+                         	var="item"
+                         	styleClass="trade-table hidden"
+                         	style="display:#{localTradeList.items.isEmpty() ? 'none;' : 'block'}">
+               	 <h:column>
+                    	<f:facet name="header">Barcode</f:facet>
+                    	#{item.barcode}
+                	</h:column>
+                	<h:column>
+                    	<f:facet name="header">Item</f:facet>
+                    	#{item.name}
+                	</h:column>
+                	<h:column>
+                		<h:commandButton
+                			id="deleteItem"
+                			value="delete"
+                			icon = "fa-trash-alt"
+                			action="#{localTradeList.deleteItem(item)}"
+                			>

```

Comment:
Place '>' at the end of the action value.

Reply:
I see how this is annoying